### PR TITLE
Some tweaks to content for the benefit of SEO keywords.

### PR DESCRIPTION
**Pull request by lewisnyman**
I've made a few SEO related tweaks to content on the site. It seemed like a good opportunity to add our names and links to us on twitter to show we are real humans. It looks a little weird right now. Any suggestions?
screenshot 2016-07-07 10 44 10

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -5,8 +5,12 @@ title: Hello, We Are Convivio
 
 Convivio is a boutique agency designing and developing digital services for innovative public and private sector organisations.
 
-That means we take clients beyond simply marketing themselves on the web, to delivering their core services online.
+That means we take clients beyond simply marketing themselves on the web, to delivering their core services online, using modern technology and processes to achieve great <br class="only-full-width" />user experiences.
 
-Our team has delivered this digital transformation before in the government, finance, and media sectors.
+We have delivered hugely successfully digital transformation before in the government, finance, and media sectors.
```
Comment (steveparks) [inline index.md]: typo - "hugely successfully" should be 'hugely successful"
However, I prefer not to use words like hugely, so could we just say successful?

File: index.md [hunk 2]
```diff
@@ -5,8 +5,12 @@ title: Hello, We Are Convivio
...
 
-This is release 1 of our website, our ‘minimum launchable product’. We're going to work in the open to develop our site, so you can see our workings-out. Follow thinking and progress on our <a href="https://blog.weareconvivio.com" target="_blank">blog.</a>
+This is release 1 of our website, our ‘minimum launchable product’. We are going to work in the open to develop our site, so you can see our workings-out and our mistakes.
+
+We are very excited on what the future will bring for Convivio. Follow us and join us <br class="only-full-width" />in our journey!
```
Comment (steveparks) [inline index.md]: typo 'excited on' should be 'excited about'
Also, that part of this sentence feels unnecessary

**Comment by steveparks** [comment]
Have made some inline comments on wording.

Additionally, as the text block is growing, is it time to pull out the bit about this being an early release of our site to a separate block? Maybe right down at the bottom, or even a floating footer?

**Comment by lewisnyman** [comment]
Updated screenshot
screenshot 2016-07-11 11 28 07

**Pull request merged**